server/models/campaign.js

```diff
@@ -23,7 +23,8 @@ const CampaignSchema = new Schema({
           "Opted out"
         ],
         default: "Draft"
-      }
+      },
+      step: { type: Schema.Types.ObjectId, ref: "Step" }
```

Comment:
I'm not sure we should have step here as prospects can be in many steps, which is recorded in the Step model. Did you intend this to show the latest step they are in maybe?

I think the prospects here at campaign level should be a unique list of the prospects in the campaign, which we show on the individual campaign page. When you click to view a step in the campaign, we will show which prospects are in that step and their status.

Reply:
Based on our discussion, we'll be using this field to keep track of each prospect's current step.

Reply:
👍